### DISCUSSION TITLE
AList功能启用无效

### DISCUSSION THREAD

**Discussion by YuyangHou1230** [coolsnowwolf/lede#9586] [Bugs / 问题反馈]
我启用alist后一直显示服务未运行。![image](https://user-images.githubusercontent.com/77433509/173884881-99ef3268-b5f5-40c9-bf6f-8b165e34943f.png)

**Comment by github-h** [comment]
一共3个文件都装齐了么
![图片](https://user-images.githubusercontent.com/49114731/173894100-b8b58a93-4026-43c0-8917-be0ccac598d8.png)

https://github.com/sbwml/openwrt-alist

**Reply by YuyangHou1230** [reply]
都有的![image](https://user-images.githubusercontent.com/77433509/173991058-42ed6989-1a97-41a7-bacb-3bf2665df204.png)

**Reply by github-h** [reply]
使用上面那个项目的ipk重装试试呢
![图片](https://user-images.githubusercontent.com/49114731/173992849-6096ae9b-88cc-4da8-af5e-a7413fb3351c.png)
